source/tests/pt/model/test_se_e2_a.py

```diff
@@ -40,9 +40,10 @@ def test_consistency(
         dstd = rng.normal(size=(self.nt, nnei, 4))
         dstd = 0.1 + np.abs(dstd)
 
-        for idt, prec in itertools.product(
+        for idt, prec, em in itertools.product(
             [False, True],
             ["float64", "float32"],
+            [[[0, 1]], [[1, 1]]],
```

Comment:
Should we keep the test for `[]` (the original case)?

Reply:
sure